src/Faker/Extension/DateTimeExtension.php

```diff
@@ -236,5 +236,5 @@ public function century(): string;
      *
      * @example 'Europe/Rome'
      */
-    public function timezone(): string;
+    public function timezone(string $countryCode = null): string;
```

Comment:
Please add this parameter, with description, to the phpdoc.

Reply:
Done!